spec/lib/voight_kampff/test_spec.rb

```diff
@@ -89,7 +77,7 @@
 
     context 'using user agent not described by custom patterns' do
       let(:user_agent_string) {
-        'DoCoMo/2.0 N905i(c100;TB;W24H16) (compatible; Googlebot-Mobile/2.1; +http://www.google.com/bot.html)'
+        'DoCoMo/2.0 N905i(c100;TB;W24H16) (compatible; Googlebot-just-kidding-Mobile/2.1; +http://www.google.com/bot.html)'
```

Comment:
This test was failing but it should've been passing. In this test we're supposed to use a user-agent not in the custom patterns config. `Googlebot-Mobile/2.1` was in the custom patterns config already, so I changed it here to be a string that's not in that config